packages/electrode-archetype-react-app-dev/lib/dev-admin/middleware.js

```diff
@@ -294,10 +295,11 @@ doReload(1); </script></body></html>`)
       );
     }
 
-    const serveStatic = (baseUrl, fileSystem, indexServer, cwd) => {
+    const serveStatic = (baseUrl, fileSystem, indexServer, cwd, isMemFs) => {
       req.originalUrl = req.url; // this is what express saves to, else serve-index nukes
-      req.url = req.url.substr(baseUrl.length) || "/";
-      const fullPath = Path.join(cwd || process.cwd(), req.url);
+      req.url = req.url.substr(baseUrl.length) || cwd;
+      const PathLib = isWin32 && isMemFs ? Path.posix : Path;
```

Comment:
again, no win32 check needed.  `isMemFs` should always use `Path.posix`.

Reply:
removed Win32 condition, using posix() directly whenever MemFS is involved irrespective of the platform.